analytics/management/commands/update_page_analytics.py

```diff
@@ -75,13 +75,13 @@ def handle(self, *args, **options):
                 '{} must be a file which contains your api key.'.format(key_file_location)
             )
 
-        scope = ['https://www.googleapis.com/auth/analytics.readonly']
+        scopes = ['https://www.googleapis.com/auth/analytics.readonly',]
```

Comment:
To be clear, square brackets, list, are alway list, no comma needed, but you can included it. Generally it isn't included if it is on one line but if you did this, I would recommonend the trailing comma:

```
values = [
    'foo',
    'bar',
]
````